projects/lua/fuzz_lua.c

```diff
@@ -147,7 +147,8 @@ LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
     if (L == NULL) {
         return 0;
     }
-    dochunk(L, luaL_loadbuffer(L, data, size, "test"));
+    const char *mode = luaL_optstring(L, 3, "bt");
```

Comment:
What's the point of using `luaL_optstring` here, given that stack is empty? Why not just pass `"bt"` directly to `luaL_loadbufferx`?

Reply:
Please check now. The point was purely from a copy-paste perspective (https://github.com/lua/lua/blob/45948e7e55c753cf0e370b251ac1d4e7f3aabedd/lbaselib.c#L373)

Reply:
Looks better now (that segment was responsible for accepting input from Lua side caller), but here's one last thing: you should actually pass `"t"` (text) instead of `"bt"` (binary or text).

Reply:
Given that there is an option "bt" aren't we actually interested in this? I mean, if it happens that there is a vulnerability due to a binary load - the Lua community must be interested in knowing this?

Reply:
Actually, no. "Binary" means bytecode in this context, which is not checked for consistency and will cause vulnerabilities and crashes by design. Lua developers aren't interested in issues caused by it.

Reply:
Perfect - thanks a lot!